src/devtools/client/debugger/src/components/SourceOutline/getOutlineSymbols.tsx

```diff
@@ -2,19 +2,34 @@ import { fuzzySearch } from "../../utils/function";
 import { groupBy, keyBy } from "lodash";
 import { SourceSymbols, ClassSymbol, FunctionSymbol } from "../../types";
 
-export function getOutlineSymbols(symbols: null | SourceSymbols, filter: string) {
+function addHitCountsToFunctions(functions, hitCounts) {
+  return functions.map(functionSymbol => {
```

Comment:
:hand: Got a `map()` that doesn't _look_ like it returns in all cases.  Does the loop always hit something, or are there times we wouldn't find anything and would return `undefined`?

Reply:
Might have written this code in 10 minutes before running to dinner 😉

Reply:
This *should* always return something. If it doesn't, then we will return `0` correctly down on line 19 👍